presto-docs/src/main/sphinx/security/kerberos-configuration.fragment

```diff
@@ -0,0 +1,24 @@
+Kerberos Configuration
```

Comment:
Probably. I've changed it and added a note at the end that if you're using a different Kerberos, you'll need to adapt the configuration.